server/src/main/java/org/opensearch/client/ClusterAdminClient.java

```diff
@@ -320,6 +322,14 @@ public interface ClusterAdminClient extends OpenSearchClient {
      */
     NodesStatsRequestBuilder prepareNodesStats(String... nodesIds);
 
+    /**
+     * QueryGroup stats of the cluster.
+     *
+     * @param request  The QueryGroupStatsRequest
+     * @param listener A listener to be notified with a result
+     */
+    void queryGroupStats(QueryGroupStatsRequest request, ActionListener<QueryGroupStatsResponse> listener);
+
```

Comment:
Why should we add this method in the ClusterAdmin interface ? I think this API should be accessible to all so does it make sense to add to `AbstractClient` since `NodeClient` extends it.

Reply:
the actual implementation is in `AbstractClient `. I see nodeStats is also following this pattern (abstract function in `ClusterAdminClient.java` and implementation in `AbstractClient`)